js/app.js

```diff
@@ -16,6 +16,6 @@ document.addEventListener('DOMContentLoaded', () => {
     )
     // For all the remaining anchors, add the external link class
     .forEach((anchor) => {
-      anchor.className = [anchor.className, 'usa-link--external'].join(' ')
+      anchor.className = [anchor.className, 'usa-link--external'].join(' ').trim();
```

Comment:
When troubleshooting this I noticed that the class list in the HTML being generated on the two links in the content timeliness standard page read `class=" usa-link--external"` with a space at the start of the class list, trim removes those.